doc/commands/group.8.rst

```diff
@@ -34,7 +34,8 @@ Description
 ===========
 
 The ``group`` command in ``DNF5`` offers several queries for getting information about groups, packages
-related to them and it is also used for groups installation.
+related to them and it is also used for groups installation. Note that some
+things dnf-4 listed as groups are now listed with ``environment list``.
```

Comment:
I think it would be useful to include that those environments used to be `groups`.

I read something about environments, but ignored that because I used to look at groups (not knowing that some groups where actual environments)

How about:
```suggestion
related to them and it is also used for groups installation.
To query environments use separate ``environment`` command.
Note: ``dnf-4`` listed both environments and groups with the ``group`` command.
```

Reply:
ok, that makes sense.